src/cascadia/TerminalApp/Pane.cpp

```diff
@@ -366,11 +368,16 @@ void Pane::_ControlWarningBellHandler(const winrt::Windows::Foundation::IInspect
     auto paneProfile = settings.FindProfile(_profile.value());
     if (paneProfile)
     {
-        if (paneProfile.BellStyle() == winrt::Microsoft::Terminal::Settings::Model::BellStyle::Audible)
+        if (WI_IsFlagSet(paneProfile.BellStyle(), winrt::Microsoft::Terminal::Settings::Model::BellStyle::Audible))
         {
             const auto soundAlias = reinterpret_cast<LPCTSTR>(SND_ALIAS_SYSTEMHAND);
             PlaySound(soundAlias, NULL, SND_ALIAS_ID | SND_ASYNC | SND_SENTRY);
         }
+        if (WI_IsAnyFlagSet(paneProfile.BellStyle(), winrt::Microsoft::Terminal::Settings::Model::BellStyle::Visual))
+        {
+            // Bubble this event up to app host, starting with bubbling to the hosting tab
+            _PaneFlashTaskbarHandlers(nullptr);

```

Comment:
I'd prefer the events to be named something like "VisualBell" or "RaiseVisualBell" -- the pane, tab, and page don't _need_ to know about a thing called a "taskbar", and the event name should be more closely related to what it _means_ rather than what it does.